img2dataset/resizer.py

```diff
@@ -167,20 +167,31 @@ def __call__(self, img_stream, blurring_bbox_list=None):
                 if max(original_height, original_width) / min(original_height, original_width) > self.max_aspect_ratio:
                     return None, None, None, None, None, "aspect ratio too large"
 
+                # check if resizer was defined
+                maybe_blur_later = False
+                if blurring_bbox_list is not None and self.blurrer is None:
```

Comment:
let's put this in the initialization instead

Reply:
I'm not sure if this can be done - maybe_blur_later is a flag that depends on the downscale parameter, blurring an image if resizing is skipped (so it is per image) while the check depends on blurring_bbox_list, in case it is provided without a blurrer in the initialization (which is also per image).

I'll change the comments to better reflect that.